src/DebugEngineHost/HostConfigurationSection.cs

```diff
@@ -42,5 +42,7 @@ public IEnumerable<string> GetValueNames()
         {
             return _key.GetValueNames();
         }
+
+        public IntPtr YOLOHandle => _key.Handle.DangerousGetHandle();
```

Comment:
>Handle [](http://example.com/codeflow?start=41&length=6)

I am fairly certain that you can pass SafeHandles in P/Invoke signatures. You might want to do that instead of using `DangerousGetHandle`.